source/configuration/modules/omhiredis.rst

```diff
@@ -1,47 +1,132 @@
 omhiredis: Redis Output Module
-==============================
+******************************
 
-**Module Name:** omhiredis
+===========================  ===========================================================================
+**Module Name:**             **omhiredis**
+**Author:**                  Brian Knox <bknox@digitalocean.com>
+===========================  ===========================================================================
 
-**Original Author:** Brian Knox <bknox@digitalocean.com>
 
-**Description**
+Purpose
+=======
 
 This module provides native support for writing to Redis,
 using the hiredis client library.
 
-**Action Parameters**
 
-Note: parameter names are case-insensitive.
+Configuration Parameters
+========================
 
-- **server**
-  Name or address of the Redis server
+.. note::
 
-- **serverport**
-  Port of the Redis server if the server is not listening on the default port.
+   Parameter names are case-insensitive.
 
-- **serverpassword**
-  Password to support authenticated redis database server to push messages
-  across networks and datacenters. Parameter is optional if not provided
-  AUTH command wont be sent to the server.
 
-- **mode**
-  Mode to run the output action in: "queue" or "publish". If not supplied, the
-  original "template" mode is used. Note due to a config parsing bug in 8.13,
-  explicitly setting this to "template" mode will result in a config parsing
-  error.
+Action Parameters
+-----------------
 
-- **template**
-  Template is required if using "template" mode.
+Server
+^^^^^^
 
-- **key**
-  Key is required if using "publish" or "queue" mode.
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
 
-- **userpush**
-  if set to on, use RPUSH instead of LPUSH, if not set or off, use LPUSH.
+   "word", "none", "no", "none"
 
+Name or address of the Redis server
 
-**Examples**
+
+ServerPort
+^^^^^^^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "integer", "6379", "no", "none"
+
+Port of the Redis server if the server is not listening on the default port.
+
+
+ServerPassword
+^^^^^^^^^^^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "word", "none", "no", "none"
+
+Password to support authenticated redis database server to push messages
+across networks and datacenters. Parameter is optional if not provided
+AUTH command wont be sent to the server.
+
+
+Mode
+^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "word", "template", "no", "none"
+
+Mode to run the output action in: "queue" or "publish". If not supplied, the
+original "template" mode is used. Note due to a config parsing bug in 8.13,
```

Comment:
Suggestion: Use either the `note` or `warning` Sphinx directives here. The former is sufficient, the latter might be more appropriate if 8.13 is still used on current LTS distros.